packages/server/graphql/intranetSchema/mutations/setNewTeamLead.ts

```diff
@@ -0,0 +1,62 @@
+import {GraphQLBoolean, GraphQLID, GraphQLNonNull} from 'graphql'
+import getRethink from '../../../database/rethinkDriver'
+import {isTeamLead, requireSU} from '../../../utils/authorization'
+import {InternalContext} from '../../graphql'
+import GraphQLEmailType from '../../types/GraphQLEmailType'
+import {getUserByEmail} from '../../../postgres/queries/getUsersByEmails'
+
+const setNewTeamLead = {
+  type: GraphQLNonNull(GraphQLBoolean),
+  description: 'Sets given team member as a team leader. ',
+  args: {
+    teamId: {
+      type: GraphQLNonNull(GraphQLID),
```

Comment:
please use `new GraphQLNonNull(...)` from now in preparation of graphql v16, see #5662